SwiftLeeds/Views/Sponsors/SponsorsView.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  SponsorsView.swift
+//  SwiftLeeds
+//
+//  Created by Muralidharan Kathiresan on 25/06/23.
+//
+
+import SwiftUI
+
+struct SponsorsView: View {
+    @Environment(\.openURL) var openURL
+    @StateObject private var viewModel = SponsorsViewModel()
+    
+    var body: some View {
+        List {
+            ForEach(viewModel.sections) { section in
```

Comment:
Need to make sure these sections are ordered in the correct order, in gas the API doesn't return them in that way

Reply:
the sections are an array and sorted in the ViewModel itself, so it will be alright here.
let me know if I missed anything